### ISSUE TITLE
[Feature request] don't support dbus, or at least make support for it an opt-in compile-time option

### ISSUE THREAD

**Issue by zetok** [MrFloya/rdm#5]
>HAS MAJOR ISSUES RELATED TO SECURITY!!

Among things. 

It's good to remember that when dbus craps out all the software that depends on it stops working.

I imagine that display manager ought to work regardless of whether dbus feels like working or not.

**Comment by 1wilkens** [comment]
Yeah I initially started adding dbus as I thought it would be required in "modern dms"(tm) but I no longer think that that is the case. The dbus crate is currently not used and could probably be removed.

The idea of an opt-in compile-time option sounds good, if I end up providing a dbus interface I will probably enable it through a cargo feature.

I will keep this open as long as dbus is still around in the codebase and not properly behind a feature.
